package.json

```diff
@@ -45,12 +47,15 @@
     "babel-preset-minify": "~0.5.1",
     "codemirror": "~5.52.0",
     "docco": "~0.8.0",
+    "dtslint": "^4.1.2",
     "highlight.js": "~9.18.1",
     "jison": "~0.4.18",
     "markdown-it": "~10.0.0",
     "puppeteer": "~3.1.0",
     "underscore": "~1.9.2",
     "webpack": "~4.42.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@types/babel-core": "^6.25.7"
```

Comment:
It’s a goal of the project to ship with no dependencies.

Reply:
@GeoffreyBooth removed the dependency in d15ab8c